ui-nextjs/components/explore/ExplainPanel.tsx

```
@@ -0,0 +1,183 @@
+
+  // Error state
+  if (error) {
+    return (
+      <div className="mb-4 flex items-center gap-3">
+        <span className="text-xs text-gray-500">
+          AI analysis unavailable &mdash; check that Ollama is running
+        </span>
+        <button
+          onClick={fetchExplanation}
+          className="text-xs px-2 py-1 rounded border border-[#555] text-gray-400 hover:text-white hover:border-gray-400 transition-colors"
+        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)